wgpu/gui/jupyter.py

```diff
@@ -69,7 +83,12 @@ def is_closed(self):
         return self._is_closed
 
     def _request_draw(self):
-        RemoteFrameBuffer.request_draw(self)
+        if not self._request_draw_timer_running:
+            now = time.perf_counter()
+            target_time = self._draw_time + 1.0 / self._max_fps
+            wait_time = max(0, target_time - now)
+            self._request_draw_timer_running = True
```

Comment:
The similarities in handling the max_fps are more similar than I anticipated. I've abstracted this away in the base class somewhat. Closing this PR in favor of a new one.